pkg/operator/workload_controller_openshiftapiserver_v311_00.go

```diff
@@ -265,6 +265,11 @@ func manageOpenShiftAPIServerConfigMap_v311_00_to_latest(kubeClient kubernetes.I
 			Namespace: targetNamespaceName,
 			Name:      "serving-cert",
 		},
+		resourcehash.ObjectReference{
```

Comment:
this is the meat here.  It calculates a hash based on content and changes to that hash will cause a config change, which will cause a rollout